week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -10,17 +10,19 @@ The output of running your code should be:
 */
 
 class ShoppingCart {
-  // Add your code here
-
-  cartContains() {
-    // Use console.log() to output everything contained in your cart
-  }
+	constructor(toiletRoll, eggs, pasta) {
+		this.toiletRoll = toiletRoll;
+		this.eggs = eggs;
+		this.pasta = pasta;
+	}
+	cartContains() {
+		console.log("Your Shopping cart has 3 Items:");
+		console.log("Toilet Roll " + this.toiletRoll);
+		console.log("Egg" + this.eggs);
+		console.log("Pasta" + this.pasta);
```

Comment:
If you had an array instead of elements then it would have been easier to iterate it over.